Oracle/shrine_export_AdapterMapping.sql

```diff
@@ -0,0 +1,13 @@
+set linesize 32000
+set pagesize 0   -- No header rows
+set trimspool on -- remove trailing blanks
+set feedback off
+-- select '"'|| SCIHLS_PATH || '","'|| HERON_PATH || '"' from  SHRINE_ONT.TEMP_SHRINE_MAPPING order by SCIHLS_PATH;
+-- some row has quatation in theri values, That is why I need to run sql statment.
+spool AdapterMappings.csv
+select '"'|| SCIHLS_PATH || '","'|| HERON_PATH || '"' from  SHRINE_ONT.TEMP_SHRINE_MAPPING
```

Comment:
Are we guaranteed that there are no double-quotes in SCILHS_PATH?

Using sqlplus to write CSV files is pretty risky. I hope we can move toward using [Ant SQL Task](https://ant.apache.org/manual/Tasks/sql.html). But I guess if this is working, I can live with it.